java/open-existing-map/README.md

```diff
@@ -1,19 +1,35 @@
-# Open Existing Map
-Open an existing map as a `PortalItem` from a `Portal`.
+# Open map URL
```

Comment:
Looks like this sample is called "open existing map"

Reply:
sorry about that!